app/routes/projects/index.tsx

```diff
@@ -627,6 +653,11 @@ export default function Projects() {
           </Grid>
         </Grid>
       </Container>
+      <MembershipModal
+        open={projectMembership.length > 0}
+        handleCloseModal={() => setShowJoinModal(false)}
```

Comment:
As we don't need the handleClose for the modal, we can do something like
```
handleCloseModal={()=> null}
```
This will have the same behavior and we will be able to remove the state variable. If we want to allow the user to close the modal from the "X" icon, then we will have to restrict the component rendering with the state variable.